content/en/docs/instrumentation/net/libraries.md

```diff
@@ -89,12 +94,12 @@ app.Run();
 When you run this code and access the `/hello` endpoint, it will:
 
 * Start a new trace
-* Automatically generate a span representing the request made to the endpoint
-* Automatically generate a child span representing the HTTP GET made to
+* Generate a span representing the request made to the endpoint
+* Generate a child span representing the HTTP GET made to
   `https://example.com/`
 
-If you add more instrumentation libraries, then you can generate more data
-automatically.
+If you add more instrumentation libraries,
+then you get more telemetry data.
```

Comment:
Similar comment as above, I don't think that clarifying that the data comes from an instrumentation library means we should drop any mention that the data gets generated automatically.

Reply:
I think it is a lot better to remove it so that it is not confused with automatic instrumentation.